cli/cmd/encore/main.go

```diff
@@ -156,7 +157,7 @@ func streamCommandOutput(stream commandOutputStream, convertJSON bool) int {
 			case st.Code() == codes.FailedPrecondition:
 				fmt.Fprintln(os.Stderr, st.Message())
 				return 1
-			case err == io.EOF || st.Code() == codes.Canceled:
+			case err == io.EOF || st.Code() == codes.Canceled || strings.HasSuffix(err.Error(), "error reading from server: EOF"):
```

Comment:
For some reason the clean `EOF` error wasn't being picked up (I think it was because it was wrapped inside a GRPC error)